lib/installer.js

```diff
@@ -299,6 +300,9 @@ var Installer = class {
       this.flow.installStep.progress.addMarker(100, 100, 10, () => {
         this.flow.finishFlow();
       });
+      if (typeof(this.onFinish) === 'function') {
```

Comment:
Not that it change anything in the behaviour, but `typeof` is kinda an operator so you can use `typeof this.onFinish` without the parenthesis.